internal/operator-controller/authorization/rbac.go

```diff
@@ -91,8 +94,14 @@ func (a *rbacPreAuthorizer) PreAuthorize(ctx context.Context, manifestManager us
 
 	for _, obj := range dm.rbacObjects() {
 		if err := ec.checkEscalation(ctx, manifestManager, obj); err != nil {
-			// In Kubernetes 1.32.2 the specialized PrivilegeEscalationError is gone.
-			// Instead, we simply collect the error.
+			missingEscalationRules, namespace := parseEscalationErrorForMissingRules(err)
+			// Check if we already have these escalation PolicyRules, if so don't append
```

Comment:
Without this check we end up with a final compacted policy rule with duplicates of the same verb over and over in the []Verbs

Reply:
Yeah, I had run into that in my PoC as well. Here's the code I added to the CompactRules function to take care of that: [`d67e50f` (#1804)](https://github.com/operator-framework/operator-controller/pull/1804/commits/d67e50f96600cdcf702c4b27967a68c357120ed1#diff-8397762fbba322656f77c15085af07a46ea1c5cdaa9e4260b737568258d6dbf8R58-R63)

Reply:
@bentito this is addressed, same topic as another thread